### ISSUE TITLE
Memento - Cancelled Call (UDP) spuriously (?) fails

### ISSUE THREAD

**Issue by rkd-msw** [Metaswitch/clearwater-live-test#108]
```
$ bundle exec rake test[staging.cw-ngv.com] (...) TESTS="MEMENTO - Cancelled Call"
Memento - Cancelled Call (TCP) - (6515550070, 6515550052) Passed
Memento - Cancelled Call (UDP) - (6515550004, 6515550060) Failed
Endpoint threw exception:
 - Call record contains wrong from_uri; found: sip:6515550075@staging.cw-ngv.com, expected: sip:6515550004@staging.cw-ngv.com
<call><to>
        <URI>sip:6515550060@staging.cw-ngv.com</URI>
</to>
<from>
        <URI>sip:6515550075@staging.cw-ngv.com</URI>
        <name>e9f79c11385fce903c16bbf8e35150ed</name>
</from>
<outgoing>1</outgoing>
<start-time>2015-07-21T10:42:17</start-time>
<answered>0</answered>

</call>
   - /home/ubuntu/clearwater-live-test/lib/tests/memento.rb:64:in `check_users_call_list'
   - /home/ubuntu/clearwater-live-test/lib/tests/memento.rb:330:in `block (2 levels) in <top (required)>'
ubuntu@ip-10-0-0-85:~/clearwater-live-test$ date
Tue Jul 21 10:53:52 UTC 2015
```

It seems to be picking up an old call from ~10 minutes ago instead of the one just made.

I can only repro this on the repo server (not my own VM) despite using the current master branch for both. Also, it doesn't repro if I run just the one test over UDP - only when that test follows some other calls.

**Comment by johadalin** [comment]
This issue appears on Rejected Call tests as well. The problem appears to probably be a test issue, rather than code, as the test only fails when there are repeats, not when numerous single instances are manually run in succession.

**Comment by rkd-msw** [comment]
Fixed by #127.
